lib/text/lrc_text_parser.js

```diff
@@ -33,16 +33,16 @@ shaka.text.LrcTextParser = class {
    * @export
    */
   parseMedia(data, time) {
-    return shaka.text.LrcTextParser.getCues_(data, time.segmentEnd);
+    return shaka.text.LrcTextParser.getCues(data, time.segmentEnd);
```

Comment:
We cannot do this because getCues is static and is necessary in the player.

Reply:
Part of my generic suggestion was to avoid checking `if (mimeType == 'foo')` and instead use the parser interface directly.  Doing something like this:

```js
const factory = TextEngine.findParser(mimeType);
if (factory) {
    const obj = factory();
    cues = obj.parseMedia(data);
}
```

But you don't have to do that now.  You could do it in another PR or I could do it instead.

Reply:
I'm going to address it in another PR, I keep it on my radar. Thanks!